package.json

```diff
@@ -41,6 +41,6 @@
     "yarn": "1.22.19"
```

Comment:
NIT: We should upgrade the yarn and node versions also

Reply:
Maybe in another PR. DX team will take care of it ;)